code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -487,23 +551,32 @@
 	color = "#60A584" // rgb: 96, 165, 132
 	taste_description = "bitterness"
 
+/datum/reagent/aranesp/on_mob_add(mob/living/L)
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.physiology.stun_mod *= 0.7
+		H.physiology.stamina_mod *= 0.7
```

Comment:
Maybe it's a good idea to put these as (static?) vars on anaresp datum, so that the values don't get desynced between `on_mob_add` and `on_mob_delete`?

Same for other chems physiology mods

Reply:
I am not quite sure I follow? 
all the `stun_mod` and `stamina_mod` etc. are multiplicative effects, similar to this one. 
so lets say they had this + blood swell, they would have 0.5 * 0.7 so 0.35. storing it as a static then replying it would cause it to become out of sync?

Reply:
A future coder can decide to buff/nerf anaresp to give, say, 0.8 stun_mod. Change the value here in `on_mob_add` but forget to change it in `on_mob_delete`. Which would make each add-remove cycle permanently buff stun_mod by `0.1`

What i suggest is something like
```dm
/datum/reagent/aranesp
	name = "Aranesp"
	id = "aranesp"
	// ...
	var/static/stun_mod = 0.7
	var/static/stamina_mod = 0.7

/datum/reagent/aranesp/on_mob_add(mob/living/L)
	if(ishuman(L))
		var/mob/living/carbon/human/H = L
		H.physiology.stun_mod *= stun_mod
		H.physiology.stamina_mod *= stamina_mod


/datum/reagent/aranesp/on_mob_delete(mob/living/M)
	if(ishuman(M))
		var/mob/living/carbon/human/H = M
		H.physiology.stun_mod /= stun_mod
		H.physiology.stamina_mod /= stamina_mod
```